README.rst

```diff
@@ -12,18 +12,36 @@ computational research data analysis workflows on remote containerised compute
 clouds. REANA was born to target the use case of particle physics analyses, but
 is applicable to any scientific discipline.
 
-
-This repository holds the REANA project web site.
+This repository holds the REANA project web site. It uses `Lektor
+<https://www.getlektor.com/>`_ static web site generator.
 
 Installation
 ------------
 
 .. code-block:: console
 
+   $ mkvirtualenv reana.io
+   $ pip install -r requirements.txt
```

Comment:
I guess you have a reason not to install lektor (and build) with docker?

Reply:
Didn't want to add Python and its dependencies into the Nginx image, but we can do that indeed... Better for reproducibility :wink: